src/main/java/org/eclipse/microprofile/starter/rest/model/ServerOptions.java

```diff
@@ -35,14 +36,26 @@
     public final MicroProfileVersion mpVersion;
     public final List<String> mpSpecs;
     public final List<JavaSEVersion> javaSEVersions;
+    public final List<String> buildTools;
 
     public ServerOptions(MicroProfileVersion mpVersion,
                          List<MicroprofileSpec> mpSpecs,
+                         List<BuildTool> buildTools,
                          List<StandaloneMPSpec> mpStandaloneSpecs,
                          List<JavaSEVersion> javaSEVersions) {
         this.mpVersion = mpVersion;
         this.mpSpecs =  mpSpecs.stream().map(spec -> spec.getCode().toUpperCase()).collect(Collectors.toList());
+        this.buildTools = buildTools.stream().map(Enum::name).collect(Collectors.toList());
         this.mpSpecs.addAll(mpStandaloneSpecs.stream().map(spec -> spec.getCode().toUpperCase()).collect(Collectors.toList()));
         this.javaSEVersions = javaSEVersions;
     }
+
+    public ServerOptions(MicroProfileVersion mpVersion,
+                         List<String> mpSpecs,
+                         List<JavaSEVersion> javaSEVersions) {
+        this.mpVersion = mpVersion;
+        this.mpSpecs =  mpSpecs;
+        this.buildTools = null;
```

Comment:
I am wondering whether it is better just to put maven there instead of `null`.

Reply:
This constructor is called for the 'old' situation where the API didn't know about buildTool. So null simulates this situation the best.